src/repository/crl.rs

```diff
@@ -544,6 +544,14 @@ impl RevokedCertificates {
             )
         ))
     }
+
+    /// Create an empty RevokedCertificates for use in SignedMessage
+    /// CMS objects - where we typically need to include a CRL, but
+    /// never need to actually revoke anything.
+    pub fn create_empty() -> Self {
```

Comment:
This should probably be called just `empty` for consistency.

Reply:
done